docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: /dev/getting-started/paper-plugins
+---
+
+# Paper Plugins
+
+Paper plugins allow developers to take advantage of more modern concepts introduced by Mojang, such as datapacks, to 
+expand the field of what the Paper API is able to introduce.
+
+:::info Experimental
+
+This is experimental and may be subject to change.
+
+:::
+
+- [Bootstrapper](#bootstrapper)
+- [Loader](#loaders)
+- [Differences](#differences)
+
+## How do I use them?
+Similarly to Bukkit plugins, you have to introduce a ``paper-plugin.yml`` file into your jar resources folder.
+Here is an example configuration.
+```yml
+name: Paper-Test-Plugin
+version: ${version}
+main: io.papermc.testplugin.TestPlugin
+description: Paper Test Plugin
+author: PaperMC
+api-version: ${apiversion}
+load: STARTUP
+bootstrapper: io.papermc.testplugin.TestPluginBootstrap
+loader: io.papermc.testplugin.TestPluginLoader
+defaultPerm: FALSE
+permissions:
+dependencies:
+```
+
+Dependency declaration is specified a bit differently than Bukkit plugins, as you are
+able to define if a dependency is required during bootstrapping. This means that any bootstrapping
+logic for your dependencies will be run before yours is run. 
+```yml
+dependencies:
+  - name: DependencyName
+    required: true
+    bootstrap: true
+```
+
+This dependency is ``required``, meaning that if this dependency is not found your plugin will fail to load.
+The bootstrap field indicates that this dependency is required during bootstrapping, which currently does not serve a purpose.
+
+## What is it used for?
+Paper plugins lay down the framework for some future API.
```

Comment:
This is really vague lol. If there's no use "yet" then at least say what the goals are?